src/components/Alerts/InfoAlert/InfoAlert.tsx

```diff
@@ -16,12 +16,12 @@ export interface InfoAlertProps {
 export const InfoAlert = ({ title, subtitle, active }: InfoAlertProps) => {
   const [closed, setClosed] = useState(false);
   const theme = useTheme();
-  const handleClose = (
-    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-  ) => {
-    event.stopPropagation();
-    setClosed(true);
-  };
+  // const handleClose = (
```

Comment:
this one is commented out, but actually not even used within the component, so maybe it could be removed already